scripts/tag-version.sh

```diff
@@ -0,0 +1,4 @@
+CURR_VERSION=$(grep -o 'version=\"[0-9.]*\"' setup.py | sed "s/version=\"\([0-9.]*\)\"/\1/")
+
+git tag "v${CURR_VERSION}" HEAD
+git push origin tag "v${CURR_VERSION}"
```

Comment:
```suggestion
CURRENT_VERSION=$(grep -o 'version=\"[0-9.]*\"' setup.py | sed "s/version=\"\([0-9.]*\)\"/\1/")

git tag "v${CURRENT_VERSION}" HEAD
git push origin tag "v${CURRENT_VERSION}"
```
If the full is just two letters more, I don't think we need to abbreviate.

Reply:
3 even, if I can count.